### PR TITLE
ebtables: include patch from Alpine to fix *-musl

### PR DESCRIPTION
**Pull request by beefcurtains**
[Original source](http://git.alpinelinux.org/cgit/aports/commit/main/ebtables/musl-fixes.patch?id=0f5076024a9700644ac9d542d2ca643fa38e77da). Thanks to Natanael Copa.

### CODE REVIEW

File: srcpkgs/ebtables/template [hunk 1]
```diff
@@ -2,28 +2,31 @@
 pkgname=ebtables
 version=2.0.10.4
 _version=${version%.*}-${version##*.}
-revision=2
+revision=3
 build_style=gnu-makefile
+conf_files="/etc/ebtables.conf"
 short_desc="Filtering tool for a Linux-based bridging firewall"
-maintainer="Enno Boland <eb@s01.de>"
+maintainer="beefcurtains <beefcurtains@users.noreply.github.com>"
```
Comment (Gottox) [inline srcpkgs/ebtables/template]: noreply addresses make no sense.
Do really you want to take over the maintainership?

**Comment by Gottox** [comment]
Thanks for the fixes! Must have been drunk when I made this template. :D

File: srcpkgs/ebtables/template (hunk 1 above)
Comment (beefcurtains) [inline srcpkgs/ebtables/template]: You actually *can* send me email on that address. However, I'd be happy to revert this change. No hard feelings, I hope?

Comment (Gottox) [inline srcpkgs/ebtables/template]: Didn't know that.

No, it's fine. I changed it back myself. :)
